docs/install.rst

```diff
@@ -1,113 +1,115 @@
 It will get you to a point of having a local running instance.
 
+Requirements
+------------
 
-First, obtain `Python 3.6`_ and virtualenv_ if you do not already have them. Using a
-virtual environment will make the installation easier, and will help to avoid
-clutter in your system-wide libraries. You will also need Git_ in order to
-clone the repository. If you plan to import Python 2.7 project to your RTD then you'll
-need to install Python 2.7 with virtualenv in your system as well.
+First, obtain `Python 3.6`_ and virtualenv_ if you do not already have them.
+Using a virtual environment is strongly recommended,
+since it will help you to avoid clutter in your system-wide libraries.
 
+Additionally Read the Docs depends on:
 
-.. _Python 3.6: http://www.python.org/
-.. _virtualenv: http://pypi.python.org/pypi/virtualenv
-.. _Git: http://git-scm.com/
-.. _Homebrew: http://brew.sh/
-.. _Elasticsearch: https://www.elastic.co/products/elasticsearch
-.. _PostgreSQL: https://www.postgresql.org/
-.. _Redis: https://redis.io/
-
-.. note::
+* `Git`_ (version >=2)
+* `Mercurial`_ (only if you need to work with mercurial repositories)
+* `Pip`_ (version >1.5)
+* `Redis`_
+* `Elasticsearch`_ (only if you want full support for searching inside the site)
 
-    Requires Git version >=2
+    * Ubuntu users could install this package by following :doc:`/custom_installs/elasticsearch`.
 
 .. note::
 
-    If you are having trouble on OS X Mavericks (or possibly other versions of
-    OS X) with building ``lxml``, you probably might need to use Homebrew_
-    to ``brew install libxml2``, and invoke the install with::
+    If you plan to import Python 2 projects to your RTD,
+    then you'll need to install Python 2 with virtualenv in your system as well.
 
-        CFLAGS=-I/usr/local/opt/libxml2/include/libxml2 \
-        LDFLAGS=-L/usr/local/opt/libxml2/lib \
-        pip install -r requirements.txt
+In order to get all the dependencies successfully installed,
+you need these libraries.
 
-.. note::
+On Mac OS
+~~~~~~~~~
 
-    Linux users may find they need to install a few additional packages
-    in order to successfully execute ``pip install -r requirements.txt``.
-    For example, a clean install of Ubuntu 14.04 LTS will require the
-    following packages::
+If you are having trouble on OS X Mavericks
+(or possibly other versions of OS X) with building ``lxml``,
+you probably might need to use Homebrew_ to ``brew install libxml2``,
+and invoke the install with::
 
-        sudo apt-get install build-essential
-        sudo apt-get install python-dev python-pip python-setuptools
-        sudo apt-get install libxml2-dev libxslt1-dev zlib1g-dev
-
-    CentOS/RHEL 7 will require::
+    CFLAGS=-I/usr/local/opt/libxml2/include/libxml2 \
+    LDFLAGS=-L/usr/local/opt/libxml2/lib \
+    pip install -r requirements.txt
 
-        sudo yum install python-devel python-pip libxml2-devel libxslt-devel
+On Ubuntu
```

Comment:
I wonder if we could use https://github.com/djungelorm/sphinx-tabs to make this a nicer UI for our users here, instead of having to scroll through all the data for platforms I don't use.

Reply:
Yeah, that will look better